### DISCUSSION TITLE
Live reload stopped working after updating to 1.19.3

### DISCUSSION THREAD

**Discussion by EmberRandall** [remix-run/remix#7429] [Q&A]
Up until a week or two ago, changing any file in the project would trigger a live reload, because we've got the `<LiveReload />` component at the root of our app. However, that recently stopped working -- changes don't take effect until an actual page refresh happens. We upgraded to 1.19.3 a few weeks ago, and that might have been the cause, but downgrading to our prior version (1.19.1) doesn't seem to fix it, so that may be a red herring.

Our remix config:
```
/** @type {import('@remix-run/dev').AppConfig} */
module.exports = {
  ignoredRouteFiles: ['**/.*', '**/*.test.ts'],
  tailwind: true,
  serverModuleFormat: 'cjs',
  future: {
    v2_dev: true,
    v2_meta: true,
    v2_routeConvention: true,
    v2_errorBoundary: true,
    v2_normalizeFormMethod: true,
    v2_headers: true
  }
}
```

I'm fairly new to remix, so I'm not sure how to debug this!

**Comment by sergiodxa** [comment]
I see you enabled `v2_dev`, this is the new compiler, did you updated your `dev` script in the package.json? Check https://remix.run/docs/en/1.19.3/pages/v2#remix-dev

**Reply by EmberRandall** [reply]
Yes, I think so? Our commands to start are:
* `remix dev -c \"yarn server:dev\" --manual`
* `yarn server:dev` calls `yarn dev-runner --ignore build --ignore app app/entrypoints/server.ts`
We've had `v2_dev` enabled for quite a while now, for what that's worth
